quizes_projects_4_OOP.py

```diff
@@ -226,3 +226,120 @@ def max_elevation_city(min_population):
 print(max_elevation_city(100000)) # Should print "Cusco, Peru"
 print(max_elevation_city(1000000)) # Should print "Sofia, Bulgaria"
 print(max_elevation_city(10000000)) # Should print ""
+
+
+
+#soln
+# define a basic city class
+class City:
+    name = ""
+    country = ""
+    elevation = 0
+    population = 0
+
+
+# create a new instance of the City class and
+# define each attribute
+city1 = City()
```

Comment:
Doesn't  Python need the new Keyword for class instances?

Reply:
Python doesn't use new unlike most languages do.

Reply:
That's very interesting